docs/docs/test-types/executor-maven.md

```diff
@@ -2,11 +2,18 @@ import Admonition from "@theme/Admonition";
 
 # Maven
 
-Testkube allows you to run Maven-based tasks which could be also tests. For example, we can easily run JUnit tests in Testkube now. 
+Testkube allows you to run Maven-based tasks which could be also tests. For example, we can easily run JUnit tests in Testkube now.
 
 * Default command for this executor: `mvn`
 * Default arguments for this executor command: `--settings` `<settingsFile>` `<goalName>` `-Duser.home` `<mavenHome>`
-(parameters in `<>` are calculated at test execution)
+
+Parameters in `<>` are calculated at test execution:
+
+* `<settingsFile>` - will be calculated based on the contents of the `--variables-file` argument; when missing, this will be skipped
+* `<goalName>` - will be set to `test` in case the test type is `maven/test`, `integration-test` in case of `maven/integration-test`, and it will be empty on test type `maven/project`
```

Comment:
```suggestion
* `<goalName>` - Will be set to `test` if the test type is `maven/test`, `integration-test` in the case of `maven/integration-test`, and it will be empty on test type `maven/project`
```